### PR TITLE
feat(service-portal): Display inbox document by id

### PR DESCRIPTION
**Pull request by thordurhhh**
## What

When visiting the postholf overview. You can pass in the id of the document in the url, it will open and display on the correct page.

## Why

New feature ✨ 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/documents/src/screens/Documents.loader.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+              pageSize: 10,
+              messageId: idParam,
+            },
+          },
+        })
+        return data?.getDocumentPageNumber.documentPage ?? 1
+      }
+      return 1
+    } catch {
+      return null
```
Comment (thorkellmani) [inline libs/service-portal/documents/src/screens/Documents.loader.ts:33]: return number?

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 2]
```diff
@@ -51,14 +51,16 @@ import {
...
+import { useGetDocumentInboxLineQueryDetailsLazyQuery } from './DocumentDetail.generated'
 
 const pageSize = 10
 
 export const ServicePortalDocuments = () => {
   useNamespaces('sp.documents')
   const userInfo = useUserInfo()
   const { formatMessage } = useLocale()
-  const [page, setPage] = useState(1)
+  const loaderNumber = useLoaderData() as number
```
Comment (thorkellmani) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:62]: scary

**Review by thorkellmani — CHANGES_REQUESTED** [review]
Nitpicks

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx (hunk 2 above)
Comment (thordurhhh) [reply libs/service-portal/documents/src/screens/Overview/Overview.tsx:61]: Yeah, that's the go to solution for the useLoaderData with typescript. In this case not so scary as we only return a number always :D

**Review by thorkellmani — COMMENTED** [review]
LGTM 🍼

**Pull request merged**